examples/fastapi/main.py

```
@@ -0,0 +1,403 @@
+"""
+FastAPI + pyhub-llm 기본 통합 예제
+
+이 파일은 pyhub-llm을 FastAPI와 연동하는 기본적인 예제를 제공합니다.
+단일 질문 처리, 배치 처리, 스트리밍 응답 등의 기능을 포함합니다.
+"""
+
+import asyncio
```

Flake8 (7.2.0)

[error] 'asyncio' imported but unused

(F401)